### PR TITLE
fix: Upgrade to v702 fails for really old version

### LINKED ISSUE #7630: bug: Upgrade to v702 fails for v301
### Describe the bug

couldn't successfully run sql_upgrade for really old version

### To Reproduce

1. Add a '....'

### Client configuration

- Browser:
- OpenEMR version:
- Operating system:

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #7630

### CODE REVIEW

**Comment by stephenwaite** [comment]
testing well

File: src/Services/Utils/SQLUpgradeService.php [hunk 1]
```diff
@@ -1025,8 +1025,9 @@ private function clickOptionsMigrate()
     private function CreateOccupationList()
     {
         $res = sqlStatement("SELECT DISTINCT occupation FROM patient_data WHERE occupation <> ''");
+        $records = [];
         while ($row = sqlFetchArray($res)) {
-            $records[] = $row['occupation'];
+            $records = $row['occupation'];
```
Comment (bradymiller) [inline src/Services/Utils/SQLUpgradeService.php:1030]: isn't this supposed to create an array of occupations?
(i think these changes will just send back the last occupation as a string)

File: src/Services/Utils/SQLUpgradeService.php [hunk 2]
```diff
@@ -1047,8 +1048,9 @@ private function CreateOccupationList()
     private function CreateReactionList()
     {
         $res = sqlStatement("SELECT DISTINCT reaction FROM lists WHERE reaction <> ''");
+        $records = [];
         while ($row = sqlFetchArray($res)) {
-            $records[] = $row['reaction'];
+            $records = $row['reaction'];
```
Comment (bradymiller) [inline src/Services/Utils/SQLUpgradeService.php:1053]: isn't this supposed to create an array of reactions?
(i think these changes will just collect the last reaction as a string)

**Comment by stephenwaite** [comment]
clapping_ape

**Comment by bradymiller** [comment]
dance-bear

**Pull request merged**